test/e2e/testing-manifests/glbc/default-svc-rc.yaml

```diff
@@ -1,16 +1,17 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: my-nginx
+  name: default-backend
 spec:
   replicas: 1
   template:
     metadata:
       labels:
-        app: nginx
+        app: default-backend
     spec:
       containers:
-      - name: nginxhttps
-        image: bprashanth/nginxhttps:1.0
+      - name: default-backend
+        image: bprashanth/echoserver:0.0
```

Comment:
will replace this container with something on gcr.io, it's just a plance holder for a 404 page